spec/factories/users.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :user, aliases: [:member] do
-    github_handle { FFaker::InternetSE.user_name_variant_short }
+    github_handle { FFaker::InternetSE.unique.user_name_variant_short }
```

Comment:
This fixes flickering specs because of failing uniqueness validation on gh handle.